tests/main/set-proxy-store/task.yaml

```diff
@@ -69,9 +79,17 @@ execute: |
     snap set core proxy.store=
     snap refresh --list | not grep -Pzq "$expected"
 
+    jq '.data.auth.device."session-macaroon"' /var/lib/snapd/state.json|NOMATCH  fake-session
```

Comment:
maybe a small comment here like: `echo "Ensure changing the proxy.store will clear out the session-macaroon" ?

Reply:
tweaked